src/Mono.Android/Android.Runtime/JNIEnv.cs

```diff
@@ -152,15 +152,16 @@ internal static unsafe void Initialize (JnienvInitializeArgs* args)
 		{
 			Logger.Categories = (LogCategories) args->logCategories;
 
-			var __start = new DateTime ();
+			Stopwatch stopper = null;
+			long elapsed, totalElapsed = 0;
 			if (Logger.LogTiming) {
-				__start = DateTime.UtcNow;
-				Logger.Log (LogLevel.Info,
-						"monodroid-timing",
-						"JNIEnv.Initialize start: " + (__start - new DateTime (1970, 1, 1)).TotalMilliseconds);
-				Logger.Log (LogLevel.Info,
-						"monodroid-timing",
-						"JNIEnv.Initialize: Logger JIT/etc. time: " + (DateTime.UtcNow - new DateTime (1970, 1, 1)).TotalMilliseconds + " [elapsed: " + (DateTime.UtcNow - __start).TotalMilliseconds + " ms]");
+				stopper = new Stopwatch ();
+				stopper.Start ();
+				Logger.Log (LogLevel.Info, "monodroid-timing", "JNIEnv.Initialize start");
+				elapsed = stopper.ElapsedMilliseconds;
```

Comment:
You should call `stopper.Stop()` before accessing `stopper.ElapsedMilliseconds`, for consistency.  (Otherwise it'll keep "running".)